app/Http/Controllers/Admin/PostController.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Post;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use \Validator;
+class PostController extends Controller
+{
+
+	 /**
+     * Returns all posts
+     *
+     * @return \Illuminate\View\View
+     */
+	public function index()
```

Comment:
Why do we need an admin specific post controller? Admins can handle stuff through the same pages as everyone else.

Reply:
admin specific post controller, is used by admins to control the posts

Reply:
kill it. I explained it in discord. Admins don't need an admin specific way to manage posts for now, especialy.